frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-details-participant-list-table/common-details-participant-list-table.component.html

```diff
@@ -25,10 +25,10 @@
         <ng-container *ngIf="rowData.segment">
           {{
             (rowData.listType?.toLowerCase() === memberTypes.INDIVIDUAL.toLowerCase()
```

Comment:
this is a lot of logic for the template, can we move this calculation to a function the component so it's a simple binding? preferably without the ternary chain, those are always hard for me to read personally.